Aztec/Classes/Libxml2/DOM/Data/ElementNode.swift

```diff
@@ -105,6 +105,12 @@ public class ElementNode: Node {
     }
 
     // MARK: - Node Queries
+    
+    public func attribute(named name: String) -> Attribute? {
+        return attributes.first { (attribute) -> Bool in
```

Comment:
No need to change anything. I'm just annoying: `(attribute) -> Bool in`, in this case, can be compressed to just `attribute in` (because the return type gets inferred)